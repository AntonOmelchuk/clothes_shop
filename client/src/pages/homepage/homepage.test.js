import React from 'react';
import { shallow } from 'enzyme';
import Homepage from './homepage.component';

it('renders Homepage component', () => {
  expect(shallow(<Homepage />)).toMatchSnapshot();
});
