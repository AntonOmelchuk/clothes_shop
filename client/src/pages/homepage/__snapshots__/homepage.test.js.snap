// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Homepage component 1`] = `ShallowWrapper {}`;

exports[`should render Homepage component 1`] = `ShallowWrapper {}`;
